nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -2660,6 +2665,19 @@
         - 'mercurial.charencode'
         - 'mercurial.cext.parsers'
 
+- module-name: 'mitmproxy' # checksum: 4eec16b2
+  implicit-imports:
+    - depends:
+        - 'mitmproxy_macos'
```

Comment:
That might be missing a `when` clause, I think, to limit it to macOS, esp. if it exists on other OSes, otherwise not important.